### ISSUE TITLE
Support transport security (iOS 9)

### ISSUE THREAD

**Issue by silyevsk** [rollout/rollout.io-ios#49]
iOS 9 blocks by default any request over http, so all the requests by Rollout SDK should be made over https.

As a temporary workaround, in case you are testing your app with iOS 9, until we move to https, the requests can be allowed via info.plist with the following configuration:

```
	<key>NSAppTransportSecurity</key>
	<dict>
		<key>NSAllowsArbitraryLoads</key>
		<true/>
			<key>rollout.io</key>
			<dict>
				<key>NSIncludesSubdomains</key>
				<true/>
				<key>NSExceptionsAllowsInsecureHTTPLoads</key>
				<true/>
			</dict>
	</dict>
```

**Comment by oprusak** [comment]
Now that you've released version 0.10.0 of your SDK which supports iOS 9, is this still needed?

**Comment by silyevsk** [comment]
Fixed in 0.10.0
